shyftdb/shyft_database_util.go

```diff
@@ -126,7 +142,7 @@ func WriteTransactions(sqldb *sql.DB, tx *types.Transaction, blockHash common.Ha
 	gas := txData.Gas
 	data := txData.Data
 	to := txData.To
-	if(to == nil){
+	if to == nil{
```

Comment:
should we be making style changes?

Reply:
Are you referring to assigning the variables before using them in line 106?